include/fcl/narrowphase/detail/primitive_shape_algorithm/box_box-inl.h

```diff
@@ -587,10 +610,8 @@ int boxBox2(
     pa += ua * alpha;
     pb += ub * beta;
 
-
-    // Vector3<S> pointInWorld((pa + pb) * 0.5);
-    // contacts.push_back(ContactPoint<S>(-normal, pointInWorld, -*depth));
-    contacts.emplace_back(normal, pb, -*depth);
+    Vector3<S> pointInWorld((pa + pb) * 0.5);
+    contacts.push_back(ContactPoint<S>(normal, pointInWorld, *depth));
```

Comment:
I think @jslee02 switched from `push_back` to `emplace_back` in 3693e16cb2ba780e0a953a18186a10a6cc957257